docs/build/build-parachains.md

```diff
@@ -130,8 +130,8 @@ good return on investment.
 parachain. They can be incentivized with a native token payout from:
 
 - Transaction fees collected
-- Parathread token sponsorship
-- Blocks are naturally produced when a parathread bid is less than the native token payout.
+- Parachain token sponsorship
+- Blocks are naturally produced when a parachain bid is less than the native token payout.
```

Comment:
This sentence does not make sense. How do collators get incentivized with parachain bid?

```suggestion
```